base/applications/mspaint/lang/bg-BG.rc

```diff
@@ -140,6 +140,10 @@ BEGIN
     "^I", IDM_IMAGEINVERTCOLORS
     "^E", IDM_IMAGEATTRIBUTES
     "N", IDM_IMAGEDELETEIMAGE, CONTROL, SHIFT, VIRTKEY
+    VK_ADD, IDM_CTRL_PLUS, CONTROL, VIRTKEY
+    VK_SUBTRACT, IDM_CTRL_MINUS, CONTROL, VIRTKEY
+    VK_OEM_PLUS, IDM_CTRL_PLUS, CONTROL, VIRTKEY
+    VK_OEM_MINUS, IDM_CTRL_MINUS, CONTROL, VIRTKEY
```

Comment:
General question: I wonder whether, when these accelerators are all the same for all languages, whether they could be placed in the language-neutral section in the main mspaint .rc file :thinking:

Reply:
The neutral accelerators are possible.
But some languages might want to get different key binding (?unconfirmed).

Reply:
If there are language-specific accelerators, they will override the neutral.

Reply:
For example, `pt-PT.rc` has line `"^O", IDM_FILENEW`.

Reply:
good catch :+1: